presto-native-execution/pom.xml

```diff
@@ -2,19 +2,20 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
-        <artifactId>presto-root</artifactId>
         <groupId>com.facebook.presto</groupId>
-        <version>0.274-SNAPSHOT</version>
+        <artifactId>presto-root</artifactId>
+        <version>0.275-SNAPSHOT</version>
```

Comment:
> curious: why is this not changed automatically by our release process? Or it will be fixed together with this PR?

I suppose that's because this component wasn't added to the root pom.xml before this PR on master.